package/run_sonobuoy_plugin.sh

```diff
@@ -59,7 +59,6 @@ mkdir -p "${RESULTS_DIR}"
 
 # etcd
 if [[ "${OVERRIDE_BENCHMARK_VERSION}" != "" ]]; then
-  if [[ "$(pgrep -f /etcd | wc -l)" -gt 0 ]]  || [[ "$(journalctl -D $JOURNAL_LOG -u k3s | grep 'Managed etcd' | grep -v grep | wc -l)" -gt 0 ]]; then
```

Comment:
This impacts all rke1/rke2/k3s or any custom benchmark scan. This check makes sure for rke1 that etcd benchmarks are run only on nodes having etcd running. So I dont think we should be deleting this out.

Reply:
This has been updated to only affect the k3s side of this check. If k3s is running we now always pass. This should limit the impact to other distributions.